client/post-editor/controller.js

```diff
@@ -81,7 +83,7 @@ function getPressThisContent( query ) {
 			ReactDomServer.renderToStaticMarkup(
 				<p>
 					<a href={ url }>
-						<img src={ image } />
+						<img src={ image } alt="" />
```

Comment:
It was a simple solution to a required lint fix; it's valid, although I could give it a slightly more helpful "featured image for the post" or something like that.

Reply:
😉 this is an example of a "fix" working around the problem instead of fixing it.

I recommend either _being bold_ and telling `eslint` "NO!" or fixing it by providing a more helpful alternate value. I'm not familiar with the code so I don't know what would be appropriate, but I'm pretty sure an empty string is not better than a missing attribute

Reply:
`alt=""` has a well-defined meaning that is different from not specifying the attribute at all: it tells the screenreader that the image is for decoration and can be safely ignored by it.

See also:
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/alt-text.md
https://www.w3.org/WAI/WCAG21/Techniques/html/H67

Reply:
`alt=""` has a well-defined meaning that is different from not specifying the attribute at all: it tells the screenreader that the image is for decoration and can be safely ignored by it.

See also:
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/alt-text.md
https://www.w3.org/WAI/WCAG21/Techniques/html/H67